lib/renderer.rb

```diff
@@ -94,54 +117,85 @@ def render(system_description, options = {})
       heading(header)
     end
 
-    do_render
+    content(system_description)
 
     @buffer += "\n" unless @buffer.empty? || @buffer.end_with?("\n\n")
 
     @buffer
   end
 
-  def render_comparison_section(description)
-    @system_description = description
-    indent { do_render }
-    @buffer += "\n" unless @buffer.empty? || @buffer.end_with?("\n\n")
-  end
-
-  def render_comparison_only_in(description, scope)
-    if description[scope]
-      puts "Only in '#{description.name}':"
-      render_comparison_section(description)
-    end
-  end
-
-  def render_comparison_common(description, scope)
-    if description[scope]
-      puts "Common to both systems:"
-      render_comparison_section(description)
-    end
-  end
-
-  def render_comparison(description1, description2, description_common, options = {})
+  # Renders the result of a comparison of two system descriptions. The arguments to that method
+  # are three SystemDescriptions:
+  #
+  # [description1] contains all elements that are only present in description1
+  # [description2] contains all elements that are only present in description2
+  # [description_common] contains all elements that are present in both descriptions
+  #
```

Comment:
The API documentation also needs to document the argument with the changed elements.